common/src/main/java/com/google/udmi/util/GeneralUtils.java

```diff
@@ -1,10 +1,21 @@
 package com.google.udmi.util;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.util.ISO8601DateFormat;
+import java.io.File;
 import java.lang.reflect.Field;
 import java.util.ArrayList;
 import java.util.List;
 
 public class GeneralUtils {
+
```

Comment:
(non action at this moment) Later let's refactor the entire codebase to use these functions.

Reply:
yup -- at this point I was just slowly starting to collect the obvious things in one place!